portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,38 +21,37 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-    private List<String> quotes;
+    private List<String> comments;
 
+    /** Add comments to Arraylist*/
     @Override
     public void init() {
-        quotes = new ArrayList<>();
-        quotes.add(
-            "A ship in port is safe, but that is not what ships are for. "
-                + "Sail out to sea and do new things. - Grace Hopper");
-        quotes.add("They told me computers could only do arithmetic. - Grace Hopper");
-        quotes.add("A ship in port is safe, but that's not what ships are built for. - Grace Hopper");
-        quotes.add("It is much easier to apologise than it is to get permission. - Grace Hopper");
-        quotes.add("If you can't give me poetry, can't you give me poetical science? - Ada Lovelace");
-        quotes.add("I am in a charming state of confusion. - Ada Lovelace");
-        quotes.add(
-            "The Analytical Engine weaves algebraic patterns, "
-                + "just as the Jacquard loom weaves flowers and leaves. - Ada Lovelace");
-        quotes.add(
-            "Sometimes it is the people no one can imagine anything of "
-                + "who do the things no one can imagine. - Alan Turing");
-        quotes.add("Those who can imagine anything, can create the impossible. - Alan Turing");
+        comments = new ArrayList<String>();
+        comments.add("Cool videos");
+        comments.add("I miss Cancun!");
+        comments.add("Very nostalgic");
     }
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String quote = quotes.get((int) (Math.random() * quotes.size()));
-        response.setContentType("text/html;");
-        response.getWriter().println(quote);
+        String json = convertToJsonUsingGson(this.comments);
+
+        //Send JSON as response
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+
+    /** Convert Arraylist to JSON using GSON library */
+    private String convertToJsonUsingGson(List<String> comments){
+        Gson gson = new Gson();
+        String json = gson.toJson(comments);
+        System.out.println(json);
```

Comment:
This is useful to have a debug statement here when you're debugging but I would either remove this.